ui/js/directives/treeherder/clonejobs.js

```diff
@@ -357,16 +368,28 @@ treeherder.directive('thCloneJobs', [
                     addJobBtnToArray(countInfo.lastJob, lastJobSelected, jobBtnArray);
                 } else {
                     // with more than 1 job for the status, add it as a count
-                    countInfo.value = countInfo.count;
-                    countInfo.title = countInfo.count + " " + countInfo.countText + " jobs in group";
-                    countInfo.btnClass = countInfo.btnClass + "-count";
-                    var jobCountBtn = $(jobGroupCountInterpolator(countInfo));
-                    showHideElement(jobCountBtn, true);
+                    var jobCountBtn = jobGroupCountInterpolator({
+                        value: countInfo.count,
+                        title: countInfo.count + " " + countInfo.countText + " jobs in group",
+                        btnClass: countInfo.btnClass + "-count",
+                        visibleClass: "filter-shown",
+                        selectedClasses: countInfo.selectedClasses
+                    });
                     jobCountBtnArray.push(jobCountBtn);
                     jobCountBtnArray.push(' ');
                 }
             });
+            var output = {
+                jobBtnArray: jobBtnArray,
+                jobCountBtnArray: jobCountBtnArray
+            };
+            return output;
```

Comment:
Let's just return `{ jobBtnArray: ..., ... }` directly instead of assigning it to a temporary.